akka-http/src/main/scala/akka/http/scaladsl/unmarshalling/Unmarshaller.scala

```diff
@@ -47,38 +49,44 @@ object Unmarshaller
 
   //#unmarshaller-creation
   /**
-   * Creates an `Unmarshaller` from the given function.
-   */
+    * Creates an `Unmarshaller` from the given function.
+    */
   def apply[A, B](f: ExecutionContext ⇒ A ⇒ Future[B]): Unmarshaller[A, B] =
     withMaterializer(ec => _ => f(ec))
 
   def withMaterializer[A, B](f: ExecutionContext ⇒ Materializer => A ⇒ Future[B]): Unmarshaller[A, B] =
     new Unmarshaller[A, B] {
       def apply(a: A)(implicit ec: ExecutionContext, materializer: Materializer) =
         try f(ec)(materializer)(a)
-        catch { case NonFatal(e) ⇒ FastFuture.failed(e) }
+        catch {
+          case NonFatal(e) ⇒ FastFuture.failed(e)
+        }
     }
 
   /**
-   * Helper for creating a synchronous `Unmarshaller` from the given function.
-   */
+    * Helper for creating a synchronous `Unmarshaller` from the given function.
+    */
   def strict[A, B](f: A ⇒ B): Unmarshaller[A, B] = Unmarshaller(_ => a ⇒ FastFuture.successful(f(a)))
 
   /**
-   * Helper for creating a "super-unmarshaller" from a sequence of "sub-unmarshallers", which are tried
-   * in the given order. The first successful unmarshalling of a "sub-unmarshallers" is the one produced by the
-   * "super-unmarshaller".
-   */
+    * Helper for creating a "super-unmarshaller" from a sequence of "sub-unmarshallers", which are tried
+    * in the given order. The first successful unmarshalling of a "sub-unmarshallers" is the one produced by the
+    * "super-unmarshaller".
+    */
```

Comment:
This is not the scaladoc style we use, please revert

Reply:
doesn't `validatePullRequest` format code properly?

Reply:
Could be that it doesn't reformat scaladoc, not sure.

Reply:
I am also surprised that scalariform doesn't update this indentation, but I just checked and indeed it doesn't.